python/huggingfaceserver/huggingfaceserver/test_model.py

```diff
@@ -40,6 +40,20 @@ def test_bert():
     assert response == {"predictions": ["paris", "france"]}
 
 
+def test_model_revision():
+    model = HuggingfaceModel("bert-base-uncased",
+                             {"model_id": "bert-base-uncased",
+                              "model_revision": "main",
+                              "tokenizer_revision": "main",
```

Comment:
Can we use some other revision because 'main' is the default ? Rest is lgtm

Reply:
Updated to a specified commit.

Reply:
Thanks!